packages/components/src/StatusLabel/StatusLabel.css

```
@@ -1,20 +1,48 @@
 :root {
-  --statusLabel-icon-diameter: 10px;
+  --labelBackgroundColor: var(--color-success--surface);
+}
+
+.success {
+  --labelBackgroundColor: var(--color-success--surface);
+}
+
+.warning {
+  --labelBackgroundColor: var(--color-warning--surface);
+}
+
+.critical {
+  --labelBackgroundColor: var(--color-critical--surface);
+}
+
+.inactive {
+  --labelBackgroundColor: var(--color-inactive--surface);
+}
+
+.informative {
+  --labelBackgroundColor: var(--color-informative--surface);
 }
 
 .statusLabelRow {
   display: flex;
+  width: fit-content;
+  padding: 0.375rem 0.625rem 0.375rem 0.5rem;
+  border-radius: 0.75rem;
   flex-direction: row;
   flex-wrap: nowrap;
-  gap: var(--space-small);
-  width: fit-content;
+  gap: 0.375rem;
+  background-color: var(--labelBackgroundColor);
+}
+
+/* Reset the <Text> line height so we can reliably get 24px tall */
+.statusLabelRow p {
+  line-height: 1;
 }
 
 .labelTextEndAligned {
   flex-direction: row-reverse;
 }
 
-.statusIndicator {
-  display: inline-flex;
-  padding-top: var(--space-minuscule);
+.statusIndicatorWrapper {
+  display: flex;
+  padding-top: 0.125rem;
```

0.125rem should = 2px

```suggestion
  padding-top: var(--space-smallest);
```

---

Unless we're using `rem` for any potential dimension changes?

---

@taylorvnoj reason I've kept it as rems is that if we ever update space-small there is a very small chance that it would still align the indicator with the center of the text label unless the text size was changed in the same proportions.

---

That makes sense!